.github/workflows/CI.yaml

```diff
@@ -351,11 +354,18 @@ jobs:
       - name: "ACTIONS: Checkout"
         uses: actions/checkout@v2
 
+      - name: "Install: Setup MSYS2 environment"
+        uses: msys2/setup-msys2@v2
+        with:
+          path-type: inherit
+          install: procps
+
       - name: "Install: rustup 1.24.3"
         run: |
           Invoke-WebRequest "https://static.rust-lang.org/rustup/archive/1.24.3/x86_64-pc-windows-msvc/rustup-init.exe" -OutFile "rustup-init.exe"
           .\rustup-init.exe -y
           rustup --version
+        shell: powershell
```

Comment:
Why do we set `powershell` explicitly if it is enabled by default?

Reply:
It's not actually: https://github.com/gear-tech/gear/pull/1486/files#diff-b268bb7dc66e8638921e223098a11eabb92b424875ec72a19d6145dc1efbe3f2R341